clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -86,6 +87,7 @@
 import static com.linkedin.venice.utils.AvroSupersetSchemaUtils.validateSubsetValueSchema;
 import static com.linkedin.venice.utils.ByteUtils.generateHumanReadableByteCountString;
 
+import com.google.common.annotations.VisibleForTesting;
```

Comment:
We don't use Guava directly, but instead prefer to set a comment if we want to

Reply:
Removed.